pkg/R/sparkRBackend.R

```diff
@@ -0,0 +1,93 @@
+# Methods to call into SparkRBackend. 
+
+
+# Returns TRUE if object is an instance of given class
+isInstanceOf <- function(jobj, className) {
+  stopifnot(class(jobj) == "jobj")
+  cls <- SparkR:::callJStatic("java.lang.Class", "forName", className)
+  SparkR:::callJMethod(cls, "isInstance", jobj)
+}
+
+# Call a Java method named methodName on the object
+# specified by objId. objId should be a "jobj" returned
+# from the SparkRBackend.
+callJMethod <- function(objId, methodName, ...) {
+  stopifnot(class(objId) == "jobj")
+  invokeJava(isStatic=FALSE, objId, methodName, ...)
+}
+
+# Call a static method on a specified className
+callJStatic <- function(className, methodName, ...) {
+  invokeJava(isStatic=TRUE, className, methodName, ...)
+}
+
+# Create a new object of the specified class name
+newJObject <- function(className, ...) {
+  invokeJava(isStatic=TRUE, className, methodName="<init>", ...)
```

Comment:
So I saw this point has been raised, but it seems possible to do this:
```
  def `<init>` = "hello"
```
Maybe use something like `__<init>` since "_" usually means reserved?